chipsec/utilcmd/acpi_cmd.py

```diff
@@ -50,49 +51,43 @@ class ACPICommand(BaseCommand):
     """
 
     def requires_driver(self):
-        # No driver required when printing the util documentation
-        if len(self.argv) < 3:
-            return False
```

Comment:
This looks like we will always need the driver even to get help information.  Do we want to make a decision about loading the driver after we call parser.parse_args?

Reply:
Please re-review.  I put the argument parser within requires_driver function, therefore if there is a problem parsing the args or help is called within the command, it will show the help screen before returning.

Reply:
Understood that it was called first but forgot that it would call sys.exit() and not return from requires_driver.  My bad.

So in the case of accepting a file as input we should not need to load the driver.  Can we handle this case in the pull request?

Reply:
Changes added, let me know what you think.